website/docs/components/application-state/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Application State
+description: An informational element that displays the current state of the application.
+caption: An informational element that displays the current state of the application.
+status: released
+links:
+  figma: https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?node-id=31113-50004
+  github: https://github.com/hashicorp/design-system/tree/main/packages/components/addon/components/hds/application-state
+previewImage: assets/illustrations/components/application-state.jpg
+navigation:
+  keywords: ['empty state', 'error state', 'message']
+---
+
+<section data-tab="Guidelines">
+  @include "partials/guidelines/overview.md"
```

Comment:
@MelSumner This should've stayed "guidelines," which aligns with the structure used in the rest of the component's documentation. Some pages have both "overview" and "guidelines"; some just have "guidelines."

_This suggestion would require that the file name is also fixed._
```suggestion
  @include "partials/guidelines/guidelines.md"
```

Reply:
@heatherlarsen The build fails when it was guidelines, but built successfully when I changed it to overview. I looked through the existing docs, and I see it both ways (I found a component that only had guidelines, another that only had overview), so maybe it's a Vercel thing. cc'ing @Dhaulagiri in case he has any insight here.

Reply:
@MelSumner Can you let me know which one only has overview? It's incorrect, if so.

Reply:
Must have just been a weird cache issue or off day for vercel. Reverted the change and everything is working as expected.